buildSrc/src/main/kotlin/ivy.script.gradle.kts

```diff
@@ -12,8 +12,8 @@ kotlin {
 }
 
 tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
-    kotlinOptions {
-        freeCompilerArgs += "-XXLanguage:+ContextReceivers"
+    compilerOptions {
```

Comment:
Why did you change this? Let's revert it if there's no special purpose

Reply:
Detekt reported that `kotlinOptions` is deprecated and suggests using the `compilerOptions` DSL

Reply:
Got it. Thanks! 💯